RUFAS/output_manager.py

```diff
@@ -351,13 +366,20 @@ def _load_txt_file_to_list(self, path: str) -> List[str]:
             If an error occurs while opening or reading the file.
 
         """
+        info_map = {"class": self.__class__.__name__,
+                    "function": self._load_txt_file_to_list.__name__,
+                    }
+        self.add_log("open_text_file", f"Attempting to open {path}.", info_map)
         try:
             with open(path) as text_file:
-                return text_file.read().splitlines()
+                list_of_elements = text_file.read().splitlines()
+                load_message = f"Successfully opened {path}, it contained {len(list_of_elements)} filter patterns."
+                self.add_log("filter_pattern_file_load_log", load_message, info_map)
+                return list_of_elements
         except Exception as e:
             raise e
 
-    def _filter_variables_pool(self, filter_patterns: List[str]) -> Dict[str, pool_element_type]:
+    def _filter_variables_pool(self, filter_patterns: List[str], input_file: str) -> Dict[str, pool_element_type]:
```

Comment:
```suggestion
    def _filter_variables_pool(self, filter_patterns: List[str], input_file_name: Optional[str]) -> Dict[str, pool_element_type]:
```
1. It is the file name, not the file object itself.
2. The function can work without it, hence optional.